### ISSUE TITLE
About pretrain code

### ISSUE THREAD

**Issue by HalvesChen** [AILab-CVC/SEED-X#5]
Hi, thanks for your greate work. I would like to know if you will release pre training code？

**Comment by geyuying** [comment]
Hi, the pretraining code is the same as the instruction tuning execpt for:
1. Change the pretrained model of "llm_seed_x.yaml" to llama2 pre-trained model; 
2. Change "_target_: src.models.mllm.seed_x.ContinuousLVLM.from_pretrained" in "agent_seed_x.yaml" to "_target_: src.models.mllm.seed_x.ContinuousLVLM" and remove "pretrained_model_path" 
3. Remove the instruction prompt in the data config for pre-training.

**Comment by geyuying** [comment]
The only difference between pre-training and instruction tuning is the pre-trained model and the data lodaer.

**Comment by HalvesChen** [comment]
Thanks~
